ui/apps/platform/src/Components/FilteredWorkflowViewSelector/FilteredWorkflowViewSelector.tsx

```diff
@@ -47,15 +47,15 @@ function FilteredWorkflowSelector({
                         alignItems={{ default: 'alignItemsCenter' }}
                     >
                         <Icon>{<CubesIcon />}</Icon>
-                        <span>{filteredWorkflowState}</span>
+                        <span>{filteredWorkflowView}</span>
                     </Flex>
                 </MenuToggle>
             )}
             shouldFocusToggleOnSelect
         >
             <SelectList style={{ width }}>
                 <SelectOption
-                    value="Application view"
+                    value="Application View"
```

Comment:
Aha, is this change an unintended backwards replacement?

If you expand following lines, the others are sentence case.

Reply:
I tried testing what happens if I changed this and forgot to change it back. Thanks for catching it!